### DISCUSSION TITLE
Gerätekommunikation gestört

### DISCUSSION THREAD

**Discussion by rudi-liebl** [OpenCCU/OpenCCU#1503] [🇩🇪 Fragen&Antworten]
Seit dem Update auf die Neu Version von Raspimatic (3.59.6.20211009) erhalt ich immer öfter die Meldnung Gerätelommuniktion gestört. Ich steuere mein Heizkörper und die Fußbodenheizung so wie die Fensterkontakte über den Raspimatic (Funkverbindug). Ich kann die Störung beheben in den ich am Wandthermostat die Temperatur ändere und dann auf den Sendeknopf drücke. Aber die meisten Änderungen im Webbrowser bringen diese Meldung. Was ist hier die Ursache und gibt es eine Lösung für das Problem?

**Comment by blackbasket** [comment]
Dann antworte ich dir hier nochmal... Jens hat ja recht...

Mir hat ein Tipp aus dem Homematic-Forum geholfen... 
ich habe es dann so gelöst: zyklische Statusmeldungen für die betroffenen Geräte deaktivieren -> übernehmen -> wieder aktivieren -> ok

Jetzt kannst du warten, bis die Geräte bedient werden oder du administrierst per Turnschuh und drückst überall mal die System-Taste...

**Reply by rudi-liebl** [reply]
Deinen Vorschlag habe ich schon durchgeführt aber ohne Erfolg. Das Problem ist nur kurzzeitig behoben. Nach der nächsten Änderung tritt das Problem wiedr auf. Ich habe das Problem ja auch erst seit den letzten Update. Zuvor tratt dies nicht auf.

**Reply by blackbasket** [reply]
Schade, bei mir hat's geholfen... vorher habe ich einen Monat gewartet, ob es sich alleine löst...

ich hatte es vor dem Update auch nie... deshalb tue ich mir mit @Baxxy13 's Tipp auch schwer...

**Reply by rudi-liebl** [reply]
Danke.
Könnte ein entfernen und ein neues Anlernen eine Lösung sein?

**Reply by blackbasket** [reply]
Ein Test ist es auf jeden Fall wert... wobei ich das, je nach Größe, erst mal auf wenige Geräte beschränken würde...

**Reply by rudi-liebl** [reply]
Ich dachte auch nur an die Geräte die Probleme bereiten. Ich werde es mal versuchen und gebe dann Rückmeldung.

**Comment by Baxxy13** [comment]
Klingt für mich wie ein Funkstörer in näherer Umgebung der Zentrale. Die Suche nach CARRIER_SENSE_LEVEL im Homematic-Forum sowie die Anzeige des CS auf der Startseite wären erste Anlaufpunkte wenn es sich bei den Geräten im HmIP-Geräte handelt.

**Reply by rudi-liebl** [reply]
Vielen Dank.
Mein CS schwankt zwischen 0% und 4%

**Reply by Baxxy13** [reply]
Hmm, sieht OK aus.

"Technisch gesehen" basieren alle RM-Firmwares ab 3.59.6.20210703 auf der CCU3 3.59.6. 
Am "Unterbau" hat sich also seit einigen Versionen nichts verändert. Die ganzen Optimierungen innerhalb RaspberryMatic sind hauptsächlich WebUI spezifisch, haben also eigentlich nichts mit der Kommunikation zu tun. 
Deswegen "hadere" ich etwas mit Aussagen wie "mit der letzten Version ging's, nun nicht mehr".

Aktuell fällt mir aber nichts weiter hilfreiches dazu ein. 
Aber ich drücke die Daumen das ein Neuanlernen das Problem behebt.

**Reply by blackbasket** [reply]
Ich stimme dir absolut zu... aber bis incl. August-Release hatte ich das Verhalten nie...

**Reply by blackbasket** [reply]
BTW: Das Verhalten ist nur auf einer von fünf Zentralen aufgetreten!
